website/src/index.html

```diff
@@ -313,59 +247,61 @@ <h2>Track 1</h2>
             </ul>
             </p>
             <p>
-              <b>How to submit:</b>
+              <b>How to submit:</b><br>
```

Comment:
I think this should be in a header (strictly speaking), like <h4> or sth

Reply:
h3 tag makes it look more prominent than the h2 tag, i.e., the heading of the section itself. The h2 tag is only adjusting for the size, which we can control through CSS. 

If you still think it should be either of h1 or h2 tag, let me know!

Reply:
we can use a <h3> tag and adjust the sizing

Reply:
The current version makes the b tag to be 1.2em in font-size and 900 font-weight. It looks fine to me. Can you run the website at your end and see if this is what you would like?